.github/workflows/ci.yaml

```diff
@@ -137,34 +131,31 @@ jobs:
           ref: ${{ github.head_ref }}
       - name: Fetch full git history
         run: git fetch --prune --unshallow
-      - uses: conda-incubator/setup-miniconda@v2
-        with:
-          python-version: ${{ matrix.PYTHON_VERSION }}
-          miniforge-variant: Mambaforge
-          miniforge-version: 4.11.0-2
-          use-mamba: true
-          environment-file: environment.yml
-          activate-environment: datajudge
-      - name: Run Integration Tests
-        shell: bash -l {0}
-        run: |
-          flit install -s
-          pytest --cov=datajudge --cov-report=xml --cov-append --backend=postgres tests/integration
-      - name: Generate code coverage report
-        uses: codecov/codecov-action@v3.1.1
-        with:
-          file: ./coverage.xml
-
+      - name: Setup conda
+        uses: mamba-org/provision-with-micromamba@v15
+        with:
+          extra-specs: |
+            python=${{ matrix.PYTHON_VERSION }}
+            sqlalchemy${{ matrix.SA_VERSION }}
+            psycopg2
```

Comment:
Installs the micromamba environment conditionally, i.e. for every backend and sqlalchemy version.
Therefore, these packages are missing in our default `environment.yml`.
We should probably create a new `environment-dev.yml` that has _all_ the dependencies or write a short comment that the dev has to install the dependencies for his/her backend himself later.